packages/bpk-react-utils/src/Portal.js

```diff
@@ -143,9 +143,9 @@ class Portal extends Component {
 
     this.portalElement = document.createElement('div');
     document.body.appendChild(this.portalElement);
-    document.addEventListener('touchstart', this.onDocumentMouseDown, false);
-    document.addEventListener('touchmove', this.onDocumentMouseMove, false);
-    document.addEventListener('touchend', this.onDocumentMouseUp, false);
+    document.addEventListener('touchstart', this.onDocumentMouseDown, { passive: true });
```

Comment:
It's a fairly new addition, see https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md

@georgegillams You need to apply the feature detection here too